src/screens/VotingResultsScreen/RecentResultsScreen.js

```diff
@@ -41,10 +113,39 @@ class RecentResultsScreen extends React.Component {
             <SafeAreaView style={styles.container}>
                 <View style={styles.titleContainer}>
                     <Animatable.Text animation='slideInRight' style={[{ flex: 0.8 }, styles.containerTitle]}>Resultados</Animatable.Text>
+                    {id && this.renderShareButton(topVotes)}
+                    {id && this.renderAboutButton()}
                 </View>
-                <View style={styles.messageContainer}>
-                    <Animatable.Text animation='fadeIn' style={styles.containerMessage}>Obrigado por votar na proposta. Iremos atualizá-lo assim que a votação estiver no ar.</Animatable.Text>
-                </View>
+                {!id && (

```

Comment:
Maybe a ternary expression? id ? (<>) : (<>)
And also maybe put it in a function

Reply:
Yes, we can put a separate function to render this part. It will be much better.